components/WordsForm.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+import Image from "next/image";
+import iconCheckmark from "../public/assets/voc-trainer_icon_checkmark.png";
+import iconEye from "../public/assets/voc-trainer_icon_eye.png";
+import { BiEdit, BiSave, BiTrash, BiXCircle } from "react-icons/bi";
+
+export default function WordsForm({ card }) {
+  return (
+    <StyledListItem>
+      <FirstColumn>
+        <p>{card.word}</p>
+        <p>{card.translation}</p>
+        <div>
+          <ViewsIcon>
```

Comment:
I guess an aria-label could be helpful for screen readers `aria-label="Number of views:"`.